ajax/libs/slabText/package.json

```diff
@@ -14,5 +14,6 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/freqdec/slabText"
-  }
+  },
+  "license": "MIT"
```

Comment:
Seems like also under `GPL-2.0`. Refer https://github.com/freqdec/slabText/issues/19 & https://github.com/freqdec/slabText/blob/master/js/jquery.slabtext.js#L1. The author just didn't create a file for it.